### DISCUSSION TITLE
Suspicion on excessive monster spawn after the "week of..."

### DISCUSSION THREAD

**Discussion by Branikolog** [ihhub/fheroes2#3679] [General]
![image](https://user-images.githubusercontent.com/55348946/121803627-3a708c00-cc4b-11eb-923c-9f871d03bdeb.png)
Probably, I'm a little bit paranoid, but I think there's too much Archmagi appeared this week. :)

**Comment by LeHerosInconnu** [comment]
Hello @Branikolog,

Your suspicion is correct: https://github.com/ihhub/fheroes2/issues/2366. :)

**Reply by Branikolog** [reply]
I was not sure, that density of appeared neutral monsters was described in that issue. Here, at my screenshot there appeared too many Magi... That caused my suspiciousness. ;)
